src/client/sandbox/event/unload.js

```diff
@@ -56,6 +57,19 @@ export default class UnloadSandbox extends SandboxBase {
         }
     }
 
+    _onDocumentClick (e) {
+        var target = e.target || e.srcElement;
+
+        if (target.tagName && target.tagName.toLowerCase() === 'a')
+            this.isFakeIEBeforeUnloadEvent = !isSupportedProtocol(nativeMethods.getAttribute.call(target, 'href'));
```

Comment:
I've already told my concerns about `isSupportedProtocol`: https://github.com/DevExpress/testcafe-hammerhead/pull/348#discussion_r49736383 We should rewrite it or **stop** using it.